react-main/src/app/components/Profile/index.js

```diff
@@ -20,4 +25,9 @@ const mapStateToProps = state => ({
   email: state.login.email
 });
 
-export default connect (mapStateToProps)(Profile);
+const mapDispatchToProps = dispatch => ({
+  goback: () => dispatch(back(dispatch)),
+  logout: () => dispatch(actionCreatorsLogout.logout(dispatch))
```

Comment:
as I wrote in the other PR, this action is unecessary, you can call the push action from here